packages/presentation/lib/src/pages/home_page/widgets/home_loaded.dart

```diff
@@ -47,10 +48,14 @@ class HomeBody extends StatelessWidget {
                           ? PageStorageKey(AppConst.trendsListKey)
                           : PageStorageKey(AppConst.anticipatedListKey),
                       gridDelegate: SliverGridDelegateWithFixedCrossAxisCount(
-                        crossAxisCount: AppConst.movieListCrossAxisCount,
+                        crossAxisCount:
+                            WidgetDisplayHelper.columnsAccordingWithWidth(
+                          context,
+                        ),
```

Comment:
smth with formatting

Reply:
here 2 variants:
- like is it (auto formatted)
-                         crossAxisCount:
                            WidgetDisplayHelper.columnsAccordingWithWidth(
                                context),
without comma after context

Reply:
got it. up2u